### ISSUE TITLE
Delete does not trigger refresh

### ISSUE THREAD

**Issue by cmeyertons** [Azure/vscode-kubernetes-tools#398]
When i delete a resource, i would expect a refresh on the pane to see my item was sucessfully deleted

**Comment by cmeyertons** [comment]
ah good point -- any possibility to expand the refresh to just a node of the tree?  so for full refresh you would request the root node, and this gives better performance on other actions?  or is this a limitation of the VS code tree view?

**Comment by itowlson** [comment]
Yes, that's what I need to look at - I think we can pass a hint in the 'tree data has changed' event but I've not looked into the scope of that yet.  Per your issue about multiple deletion, we need to make the post-delete refresh really fast (like, entirely local, no kubectl call), otherwise multiple delete is going to get even more frustrating as you delete, wait for the kubectl call, delete again...
